gui/storage/forms.py

```diff
@@ -1044,6 +1044,13 @@ class DiskFormPartial(MiddlewareModelForm, ModelForm):
         required=False,
     )
 
+    disk_reset_password = forms.BooleanField(
+        label='Reset Password',
+        required=False,
+        initial=False,
+        help_text=_('Click this box to reset SED password'),
+    )
+
```

Comment:
Can we disable password field when this one is checked? There are other places in the UI doing similar things you can get as an example.